### ISSUE TITLE
npm Repository Not Updated

### ISSUE THREAD

**Issue by 7erra** [MarkerAnn/health-calculator#8]
Hi!

I wanted to use this library but the npm package seems not to be up-to-date, at least in regards to the capitalization bug you fixed with 9f6177e5e3a947a50abbf9a9aa7b2220bb89043e. On Linux you get the following error when starting with Vite on Linux:
```
✘ [ERROR] Could not read from file: /home/pg/git/website/frontend/node_modules/body-measurements/dist/utils/unitConverter.js

    node_modules/body-measurements/dist/factories/HealthCalculatorFactory.js:12:36:
      12 │ import { convertUserToMetric } from '../utils/unitConverter.js';
         ╵                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~

/home/pg/git/website/frontend/node_modules/esbuild/lib/main.js:1472
  let error = new Error(text);
              ^

Error: Build failed with 1 error:
node_modules/body-measurements/dist/factories/HealthCalculatorFactory.js:12:36: ERROR: Could not read from file: /home/pg/git/website/frontend/node_modules/body-measurements/dist/utils/unitConverter.js
    at failureErrorWithLog (/home/pg/git/website/frontend/node_modules/esbuild/lib/main.js:1472:15)
    at /home/pg/git/website/frontend/node_modules/esbuild/lib/main.js:945:25
    at /home/pg/git/website/frontend/node_modules/esbuild/lib/main.js:1353:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  errors: [Getter/Setter],
  warnings: [Getter/Setter]
}

Node.js v20.18.0
```

Thanks!
7erra

**Comment by MarkerAnn** [comment]
Thank you for reaching out, I really appreciate it. I’ll take care of this first thing when I get home tonight.
/Angelica

**Comment by MarkerAnn** [comment]
I’ve updated the unitConverter.js file naming by rebuilding the project, to resolve the case sensitivity issue on Linux systems. I’ve tested the changes using Docker and published the fix in version 1.1.2 on npm. Please update to this version and let me know if any issues persist! Thanks.
